runtime/drivers/clickhouse/information_schema.go

```diff
@@ -18,46 +19,67 @@ func (c *connection) InformationSchema() drivers.InformationSchema {
 }
 
 func (i informationSchema) All(ctx context.Context) ([]*drivers.Table, error) {
-	q := `
+	conn, release, err := i.c.acquireMetaConn(ctx)
+	if err != nil {
+		return nil, err
+	}
+	defer func() { _ = release() }()
+
+	var databases []string
+	dbs, ok := i.c.config["databases"].(string)
+	if ok {
+		databases = strings.Split(dbs, ",")
+	} else {
```

Comment:
1. It feels weird to ask the user to explicitly pass a comma-separated list of database names like this. Could we just just always list the contents of all (non-system) databases/schemas?
2. Config properties should be parsed using `mapstructure` and added to the connector spec, not read and cast directly from the map

Reply:
1. I felt it gives some control to the user. There will always be cases when user passes a database in their connection string and would just be interested in looking at the tables of the passed database vs all tables in all databases. May be we can keep it a boolean config like `scan_all_databases` with default value as `true`?
2. Ack.

Reply:
A boolean would be better, but I would suggest not doing it at all for now and waiting until someone requests it. We may also want to handle it differently, e.g. by collapsing by db/schema name in the UI instead of pushing it as a config key.